docs/output/targeted.md

```diff
@@ -0,0 +1,238 @@
+
+</details>
+
+[FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) gives general quality metrics about your sequenced reads. It provides information about the quality score distribution across your reads, per base sequence content (%A/T/G/C), adapter contamination and overrepresented sequences. For further reading and documentation see the [FastQC help pages](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/).
+
+![MultiQC - FastQC sequence counts plot](../images/mqc_fastqc_counts.png)
+
+![MultiQC - FastQC mean quality scores plot](../images/mqc_fastqc_quality.png)
+
+![MultiQC - FastQC adapter content plot](../images/mqc_fastqc_adapter.png)
+
+:::info
+The FastQC plots displayed in the MultiQC report shows _untrimmed_ reads. They may contain adapter sequence and potentially regions with low quality.
+:::
+
+### Adapters
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `preprocessing/adapters/`
+  - `*_overrepresented.fasta`: Contains overrepresented sequences found by FastQC
+
+</details>
+
+[FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) finds over-represented sequences in samples. It lists all of the sequence which make up more than 0.1% of the total reads. For each over-represented sequence the program will look for matches in a database of common contaminants and will report the best hit it finds. Hits must be at least 20bp in length and have no more than 1 mismatch.
+
+### Cutadapt
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `preprocessing/cutadapt/`
+  - `*.cutadapt.log`: Cutadapt log file
+  - `*.trim.fastq.gz`: Sample reads trimmed with overrepresented sequences removed
+
+</details>
+
+### Seqtk
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `preprocessing/seqtk/`
+  - `*.seqtk-seq.fastq.gz`: Quality filtered reads.
+
+</details>
+
+[Seqtk](https://github.com/lh3/seqtk) masks (converts to Ns) bases with quality lower than 20 and removes sequences shorter than 80 bases.
+
+<!-- ### UMI -->
+
+## Mapping
+
+### minimap2
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `minimap2/`
+  - `*.bam`: BAM file containing aligned reads
+  - `*.bai`: BAI index
+
+</details>
+
+[Minimap2](https://github.com/lh3/minimap2) is a sequence alignment program that aligns DNA sequences against a reference database.
+
+### BWA
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `bwa/`
+  - `*.bam`: BAM file containing aligned reads
+  - `*.bai`: BAI index
+
+</details>
+
+[BWA-MEM](https://github.com/lh3/bwa) BWA is a software package for mapping low-divergent sequences against a reference genome.
+
+### bowtie2
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `bowtie2/`
+  - `*.bam`: BAM file containing aligned reads
+  - `*.bai`: BAI index
+
+</details>
+
+[Bowtie2](https://bowtie-bio.sourceforge.net/bowtie2/index.shtml) aligns sequencing reads to reference sequences.
+
+## Edits calling
+
```

Comment:
Could you add a description here? What does this mean?

Reply:
done